subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -39,6 +39,7 @@ static uint8_t mcuboot_buf[CONFIG_FOTA_DOWNLOAD_MCUBOOT_FLASH_BUF_SZ] __aligned(
 #endif
 static enum dfu_target_image_type img_type;
 static bool first_fragment;
+static bool download_started;
```

Comment:
Would `downloading` be a better name? You could say that even though you have stopped a download, you have still started it, it has just also been stopped :smile: